api/python/quilt3/main.py

```diff
@@ -3,15 +3,15 @@
 """
 
 import argparse
-import dns.resolver
 import subprocess
 import sys
 
+import dns.resolver
```

Comment:
Why did you move it? It's a built-it package, right?

Reply:
No, I think it's an extra lib, though one we already depended on.

Reply:
We need to add it to `setup.py`